stock_picking_batch_creation/wizards/make_picking_batch.py

```diff
@@ -156,6 +167,11 @@ def _create_batch(self, raise_if_not_possible=False):
         vals = self._create_batch_values(user, device, selected_pickings)
         batch = self.env["stock.picking.wave"].create(vals)
         batch._init_wave_info()
```

Comment:
Why do you recompute the `wave_nbr_bins` information here and not pass the value to the `_init_wave_info(wave_nbr_bins)`

Reply:
@lmarion-source The number of bins should be passed to the `init_wave_info` method